tests/Mono.Android-Tests/Directory.Build.targets

```diff
@@ -33,4 +33,52 @@
       DependsOnTargets="_CreateJavaInteropDllConfigs">
   </Target>
 
+  <!-- When we build the multitargeted Java.Interop project as a dependency of this test project we hit the following error:
+  error NETSDK1148: Found multiple publish output files with the same relative path: external\Java.Interop\bin\Release\java-interop.jar, \external\Java.Interop\bin\Release-netcoreapp3.1\java-interop.jar.
+    We do not need to include any of this content in the output directory, so remove it here:
+  -->
+  <Target Name="RemoveJavaInteropJarContent"
+      Condition=" '$(TargetFramework)' == 'net6.0-android' "
+      BeforeTargets="ComputeResolvedFilesToPublishList" >
+    <ItemGroup>
+      <ResolvedFileToPublish Remove="$(JavaInteropSourceDirectory)\bin\**\java-interop.jar" />
+    </ItemGroup>
+  </Target>
```

Comment:
This Target addition was the only change required after upgrading to .NET 6, to work around a new check that was added: https://github.com/dotnet/sdk/blob/955c0fc7b06e2fa34bacd076ed39f61e4fb61716/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ConflictResolution.targets#L97

Reply:
I think this is fine for now, can you file an issue and assign to me?

We might need to do something to clear out `@(Content)` and give a build warning. I think customers might assume these files would make it their `.apk`, and they do not.

Reply:
I think we already have some sort of check here, I filed this for follow up outside of this PR: https://github.com/xamarin/xamarin-android/issues/5299